### ISSUE TITLE
what if my audio is shorter than 30s

### ISSUE THREAD

**Issue by jidanhuang** [chavinlo/musicgen_trainer#7]
hello, I try to finetune the model on a dataset in which most of the samples are shorter than 30s, will padding the wav to 30s works? Or can I just use the original wav?

**Comment by chavinlo** [comment]
Yes the padding will work. But remember that the trainer currently is broken.
Meta should release their trainer around early-mid july

**Comment by jidanhuang** [comment]
Thank you for you response. But I just wonder why the trainer does not work since it can overfit. Early-mid july seems a little late for me.
